src/protocols/kubernetes/argv.c

```diff
@@ -67,20 +68,20 @@ void* guac_kubernetes_send_current_argv(guac_user* user, void* data) {
 
     /* Send current color scheme */
     guac_user_stream_argv(user, user->socket, "text/plain",
-            GUAC_KUBERNETES_ARGV_COLOR_SCHEME, terminal->color_scheme);
+            GUAC_KUBERNETES_ARGV_COLOR_SCHEME,
+            guac_terminal_color_scheme(terminal));
```

Comment:
For consistency with things like `guac_terminal_get_rows()`, I think this (and anything similar) should be renamed to `guac_terminal_get_color_scheme()`.

Reply:
Works for me.